lib/Round.js

```diff
@@ -1,22 +1,28 @@
 class Round {
-  constructor() {
-    this.currentRound = 1, 
+  constructor(round) {
+    this.currentRound = round, 
     this.roundOver = false, 
     this.puzzledSolved = false,
     this.winner = ''
+    this.wheel = [];
   }
 
   getPuzzle() {
 
   } 
 
-  getWheel() {
-
-  } 
+  getRoundWheel() {
+    this.wheel = new Wheel(this.currentRound);
+  }
 
   declareRoundWinner() {
 
   }
+
+  spinWheel() {
+    let randomIndex = Math.floor(Math.random() * this.wheel.length);
+    return this.wheel[randomIndex];
```

Comment:
I know this is not fully fleshed out - but where are you planning on calling the value of `spinWheel`? Curious as to what your plan is for where you will be returning this value.

Reply:
spinning the wheel is one of those activities that seem very confusing right now. We don't really have a solid understanding of where this method should live.